src/main/java/ca/sait/vezorla/controller/CustomerRestController.java

```diff
@@ -384,7 +386,9 @@ public boolean applyDiscount() {
     }
 
     @GetMapping("account/forgotpassword/{email}")
-    public void forgotPassword(@PathVariable String email) {
+    public boolean forgotPassword(@PathVariable String email) throws InvalidInputException {
```

Comment:
- Please move the `forgotPassword` functionality to `ClientRestController`
    - Customers do not have an account
- Instead of a `PathVariable`, we will be receiving the email in the `RequestBody`
    - This is done for security and to generalize the URLs